content/en/docs/setup/upgrade/index.md

```diff
@@ -90,6 +92,30 @@ $ istioctl proxy-config endpoints ${pod_name}.test-ns --cluster xds-grpc -ojson
 
 The output confirms that the pod is using `istiod-canary` revision of the control plane.
 
+### Uninstall old control plane
```

Comment:
This needs more explanation of how the different options (-f, --revision, --purge) differ and what we recommend in which situation.

Reply:
one question is should we put that in the upgrade guide or the istioctl guide

Reply:
This section is only about upgrade, so it should only talk about uninstalling an old revsion

Reply:
ok so it would only talk about `istioctl uninstall --revision` in the upgrade doc here, I would add description of other options in the istioctl guide(--purge is added already)